src/status_im/contexts/wallet/send/select_address/view.cljs

```diff
@@ -128,20 +127,13 @@
 
 (defn- f-view
   []
-  (let [on-close       (fn []
-                         (rf/dispatch [:wallet/clean-scanned-address])
-                         (rf/dispatch [:wallet/clean-local-suggestions])
-                         (rf/dispatch [:wallet/clean-selected-token])
-                         (rf/dispatch [:wallet/clean-selected-collectible])
-                         (rf/dispatch [:wallet/clean-send-address])
-                         (rf/dispatch [:wallet/select-address-tab nil])
```

Comment:
IMO, this logic should remain the same (I am not saying this could not be improved, but at least not be modified in this PR which should only modify navigation stuff).

Reply:
The logic is still the same, I just moved them to the navigator 🤔